sql/ShardSet.go

```diff
@@ -17,6 +17,7 @@ type ShardSet struct {
 	password        string
 	host            string
 	port            int
+	tls             string
```

Comment:
Should this be a `bool` instead (converted at the point of creating `ConnVariables`)? Otherwise, as a user of the library, it's pretty unclear what values I can pass to `tls`.

Reply:
Unfortunately this has to be a string b/c we need to be able to pass in `skip-verify` as a value. More info here: https://github.com/go-sql-driver/mysql#tls

Reply:
Ah, thanks for explaining. I think it's worth adding a comment to that effect. (If we were planning to maintain fun-go I'd suggest creating an enum, but that's overkill.)